src/components/ProjectSearch/TextSearch.css

```diff
@@ -0,0 +1,18 @@
+.project-text-search .result {
+  display: flex;
+  margin-left: 20px;
+}
+
+.project-text-search .file-result {
+  font-weight: bold;
+  margin-top: 20px;
+  line-height: 20px;
+}
+
+.project-text-search .line-match {
+  font-family: monospace
+}
+
+.project-text-search .result .line-number {
```

Comment:
this is getting pretty specific. maybe we could use BEM to keep this from going too far. maybe something like
`.project-text-search__line-number`. Also result seems like a bit of a dangerous css selector, as I can see it being used in a lot of places for different purposes.